### DISCUSSION TITLE
starstruck achievement

### DISCUSSION THREAD

**Discussion by XakimovNosirbek** [Schweinepriester/github-profile-achievements#177] [Q&A] [answered]
Hello! I've noticed something is not okay with the starstruck achievement. even though i have a repository with 17 stars i am not getting achievement.
<img width="1066" alt="starstark" src="https://user-images.githubusercontent.com/68727555/179310423-015637af-a4fc-448b-bf1c-8e0c4585b9b2.png">

**Comment by Schweinepriester** [comment, marked as answer]
… but it's right there, in the screenshot even?

=> https://github.com/Xakimov1610?achievement=starstruck&tab=achievements

![image](https://user-images.githubusercontent.com/2644614/179310645-8b36c4d5-966f-4735-b3a3-e3a913f25250.png)

says for that repo as well

**Comment by Chameleon-Coder** [comment]
But you can literally see it.

**Reply by hotcoding85** [reply]
please make this one:
https://github.com/hotcoding85/ckeditor5
btw i've done with yours....

**Comment by ArielMAJ** [comment]
@Xakimov1610, @Chameleon-Coder, @Schweinepriester could you guys [leave me a star](https://github.com/ArielMAJ/test.me), please? I'm almost getting this achievement.

**Reply by hotcoding85** [reply]
please make this one:
https://github.com/hotcoding85/ckeditor5
btw i've done with yours....

**Comment by hotcoding85** [comment]
please make this one:
https://github.com/hotcoding85/ckeditor5
btw i've done with yours....

**Comment by peterbaccaro** [comment]
I have created a repo that provides practical examples in Python for each of the renowned Gang of Four (GoF) design patterns.
These patterns provide proven solutions to common design challenges.
Each design pattern in this repository is accompanied by an explanation and example code.

https://github.com/peterbaccaro/design-patterns-python

Please give me a star for my repo and help me get ⭐️ starstruck ⭐️!

Thanks in advance.

**Comment by felizardo27** [comment]
Hi, this is my https://github.com/felizardo27/pokedex-RN

please help me
